src/Scripting/CSharp/project.json

```diff
@@ -5,8 +5,8 @@
     "System.Dynamic.Runtime": "4.0.10"
   },
   "frameworks": {
-    "dotnet5.4": {
-      "imports": "portable-net452"
+    "netstandard1.3": {
+      "imports": [ "portable-net452", "dotnet" ]
```

Comment:
@weshaggard @ericstj - can you just import "dotnet" and not have a version? i.e. "dotnet5.4"